spark/src/main/scala/filodb.spark/FiloRelation.scala

```diff
@@ -92,6 +95,15 @@ object FiloRelation extends StrictLogging {
     }
   }
 
+  /**
+    * Method to get possible combinations of key values provided in partition key filters.
+    * For example filters provided in the query are actor2Code IN ('JPN', 'KHM') and year = 1979
+    * then this method returns all combinations like ('JPN',1979) , ('KHM',1979) which represents
+    * full partition key value.
+    * @param xs
```

Comment:
I think you can remove this @param thing.... it's obvious from rest of description